src/main/java/HW22DB/Lesson.java

```diff
@@ -0,0 +1,13 @@
+package HW22DB;
+
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class Lesson {
+    private Integer id;
+    private String name1;
+    private Date updatedAt;
+    private Integer homework_id;
+}
```

Comment:
Entity relationship is missing between Lesson and Homework.

For example: a Lesson can have multiple Homeworks, and other way around a homework can belong only to one lesson.
Or one-to-one Lesson-Homework relationship

Consider adding a field like:
`private Homework homework;`